torch/_dynamo/bytecode_transformation.py

```diff
@@ -323,15 +427,20 @@ def transform_code_object(code, transformations, safe=False):
         "co_varnames",
         "co_filename",
         "co_name",
+        "co_qualname",  # python 3.11+
         "co_firstlineno",
         "co_lnotab",  # changed to "co_linetable" if python 3.10+
+        "co_exceptiontable",  # python 3.11+
         "co_freevars",
         "co_cellvars",
     ]
     if sys.version_info < (3, 8):
         keys.pop(1)
     if sys.version_info >= (3, 10):
         keys = list(map(lambda x: x.replace("co_lnotab", "co_linetable"), keys))
+    if sys.version_info < (3, 11):
+        keys.remove("co_qualname")
+        keys.remove("co_exceptiontable")
```

Comment:
ugh

Reply:
I guess that follows what we do for 3.8 but I agree that adding is easier to read for me as there is a single place of truth.